main.go

```diff
@@ -19,7 +19,8 @@ var (
 	redisPassword    = flag.String("redis.password", getEnv("REDIS_PASSWORD", ""), "Password for one or more redis nodes, separated by separator")
 	redisAlias       = flag.String("redis.alias", getEnv("REDIS_ALIAS", ""), "Redis instance alias for one or more redis nodes, separated by separator")
 	namespace        = flag.String("namespace", "redis", "Namespace for metrics")
-	checkKeys        = flag.String("check-keys", "", "Comma separated list of keys to export value and length/size")
+	checkKeys        = flag.String("check-keys", "", "Comma separated list of key-patterns to export value and length/size, searched for with SCAN")
+	checkSingleKeys  = flag.String("check-single-keys", "", "Comma separated list of single keys to export value and length/size")
```

Comment:
Can you please add the new flag to the README?

Reply:
Oops, sure thing.  Updated.

Reply:
Thx!